### PR TITLE
[PKG-4870] keplergl 0.3.2 ❄️

### PR DESCRIPTION
**Pull request by psteyer**
keplergl 0.3.2 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-4870]
- [Upstream repository](https://github.com/keplergl/kepler.gl/tree/v0.3.2-jupyter/bindings/kepler.gl-jupyter)

### Explanation of changes:

- Skip s390x due to missing `fiona`
- Dropped a few unused dependencies from host.
- No tests available for Python bindings.

[PKG-4870]: https://anaconda.atlassian.net/browse/PKG-4870?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by skupr-anaconda** [comment]
It looks like v0.3.2 of the package can work only with jupyterlab v3, see https://github.com/keplergl/kepler.gl/tree/v0.3.2-jupyter/bindings/kepler.gl-jupyter#jupyterlab-3 and https://github.com/keplergl/kepler.gl/blob/v0.3.2-jupyter/bindings/kepler.gl-jupyter/pyproject.toml#L2. 
I wonder if it works with jupyterlab v4 too.

**Review by skupr-anaconda — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,32 +10,33 @@ source:
...
+  skip: true  # [py<37 or s390x]
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - setuptools
     - pip
-    - jupyter
     - jupyter-packaging
```
Comment (boldorider4) [inline recipe/meta.yaml]:
```suggestion
    - jupyter-packaging >=0.7.0,<0.8.0
```

Comment (boldorider4) [reply recipe/meta.yaml]: https://github.com/keplergl/kepler.gl/blob/3e3d1631e30441bea1123d51588a1956a8402467/bindings/kepler.gl-jupyter/pyproject.toml#L2C31-L2C38

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,32 +10,33 @@ source:
...
+    - python
+    - ipywidgets >=7.0.0,<8
     - traittypes >=0.2.1
-    - pandas >=1.3.1
-    - geopandas >=0.9.0
-    - shapely >=1.7.1
-
+    - pandas >=0.23.0
+    - geopandas >=0.5.0
+    - shapely >=1.6.4.post2
```
Comment (boldorider4) [inline recipe/meta.yaml:31]:
```suggestion
    - shapely >=1.6.4.post2
    
  run_constrained:
    - jupyterlab >=3.0.0
```

Comment (boldorider4) [reply recipe/meta.yaml:31]: as pointed out in other review comments.

**Pull request merged**